gc_configs/freiburg.conf

```diff
@@ -4,23 +4,24 @@ backend = condor
 ;;backend = cream
 ;;cmdargs = -G -c -m 0
 workdir create = True
-workdir = __set__
+workdir = {workdir}
 
 [jobs]
 in flight = 1000
 wall time = 05:00:00
 memory = 3500
-jobs = __set__
+{jobs}
```

Comment:
why is this different than the other arguments ?

Reply:
not `jobs = {jobs}`

Reply:
In the first step of the simulation chain we set this option `jobs` to limit the number of events per job. Afterward we don't need this setting anymore, as one file per job is used for the rest of the simulation processing chain.